apps/web/src/utils/date.ts

```diff
@@ -32,8 +32,35 @@ export function formatDate(
   return date;
 }
 
-export function formatTtl(ttl: number) {
-  return dayjs().to(dayjs().add(ttl, "second"), true);
+export function formatTtl(seconds: number) {
```

Comment:
A comment giving credits to bee-dashboard and linking to https://github.com/ethersphere/bee-dashboard/blob/master/src/utils/index.ts#L175 would be nice here.

Reply:
Done [6d85b8a](https://github.com/Blobscan/blobscan/pull/503/commits/6d85b8a6cb16c56c4303a1c36ebcaa6e72acc4e9)